project/NativeImage.scala

```diff
@@ -268,6 +268,25 @@ object NativeImage {
         throw new RuntimeException("Native Image build failed")
       }
       log.info(s"$targetLoc native image build successful.")
+      if (targetDir != null) {
+        val symlinkTargetFile = artifactFile(null, name)
+        if (symlinkTargetFile.exists()) {
+          symlinkTargetFile.delete()
+        }
+        try {
+          val res = Files.createSymbolicLink(
+            symlinkTargetFile.toPath(),
+            targetLoc.toPath()
+          )
+          log.info(s"Symlink $res to $targetLoc created.")
```

Comment:
That should be `debug`. We print a lot of noise.

Reply:
I merged two `info` messages into one in f810615495